### PR TITLE
extract "live" blog indicator into its own component

### PR DESCRIPTION
**Pull request by bjfletcher**
This is mainly to make it easier to add the new Premium label by simplifying the Title component.  I've also updated the demo page to not use tag, image and standout. This is because that's how they're used on Next.

Before:
- Article renders Title
- Title renders the "Live" indicator and the title
- both "Live" and the title are clickable

Now:
- Article renders Blog and Title separately
- only the title is clickable

The design remains the same:

image

### CODE REVIEW

File: components/title/title.js [hunk 1]
```diff
@@ -1,31 +1,14 @@
...
-	inprogress: 'in-progress',
-	comingsoon: 'coming-soon',
-	closed: 'closed'
-};
-
 /**
  * @param {string} title
  * @param {string} url
- * @param {string} [liveBlogStatus] - inprogress|commingsoon|closed
+ * @param {boolean} [isPremium = false]
```
Comment (ironsidevsquincy) [inline components/title/title.js]: what's this for?

Comment (bjfletcher) [inline components/title/title.js]: Bad cherry picking job - removing that.

File: components/blog/blog.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const cssModifiers = {
```
Comment (ironsidevsquincy) [inline components/blog/blog.js]: not sure I'd change this name to `cssModifiers` - ties it very strongly to CSS, whereas these could be used for other reasons

**Comment by ironsidevsquincy** [comment]
Makes sense. Guess it's not a biggie losing the click area

**Comment by ironsidevsquincy** [comment]
Does JS run on those demo pages (the article time isn't being relativised)

👍

**Comment by bjfletcher** [comment]
@ironsidevsquincy great idea re initialising `o-date` on this demo page

**Pull request merged**